packages/std/src/query/wasm.rs

```diff
@@ -21,4 +21,21 @@ pub enum WasmQuery {
         /// Key is the raw key used in the contracts Storage
         key: Binary,
     },
+    /// returns a ContractInfoResponse with metadata on the contract from the runtime
+    ContractInfo { contract_addr: String },
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
```

Comment:
```suggestion
#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
#[non_exhaustive]
```

Reply:
So we can add the created_at later? Sure.

Reply:
Right, or any other field